MIGRATION.md

```diff
@@ -454,6 +458,59 @@ const isCollapsed = true;
 />;
 ```
 
+## DateTime
+
+The API of the DateTime component hasn't changed. The `format` property is the only one that was updated: from `"FULL"`, `"DATE_ONLY"`, `"TIME_ONLY"`, `"WEEKDAY_DATE"` to `"full"`, `"day"`, `"time"`, `"weekday"`.
+
+```tsx static=true
+import { DateTime } from '@contentful/f36-datetime';
+
+<DateTime date="2020-08-17T15:45:00" />
+<DateTime date="2020-08-17T15:45:00" format="day" />
+<DateTime date="2020-08-17T15:45:00" format="weekday" />
+<DateTime date="2020-08-17T15:45:00" format="time" />
+```
+
+### How to migrate your DateTime components
+
+Update the `format` value and import. It changes from this:
```

Comment:
For consistency throughout the documentation should we phrase this as follows?

To migrate your DateTime component to v4, you must update the `format` value and import. It changes from this: